elastic/system/ksp-audit-elasticsearch-log-file-access.yaml

```diff
@@ -0,0 +1,20 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-elasticsearch-log-file-access
+  namespace: default # Change your name space
+spec:
+  tags: ["ELK", "Elastic Search", "Log Access"]
```

Comment:
The tags are all over the place. Here the tag is "Elastic Search", in the earlier policy it is "Elasticsearch".
@raviknox, @vishnusomank, the way in which we are handling tags needs to be clarified to the team.